kernel/kernel-api/src/main/java/io/delta/kernel/internal/replay/LogReplay.java

```diff
@@ -414,11 +415,15 @@ private Map<String, DomainMetadata> loadDomainMetadataMap(Engine engine) {
                 snapshotHint.map(SnapshotHint::getVersion).orElse(-1L) + 1,
                 logSegment.getCheckpointVersionOpt().orElse(0L),
                 // Only find the CRC within 100 versions.
-                snapshotVersion - 100,
-                0L));
+                snapshotVersion - 100));
```

Comment:
do we still need this `crcSearchLowerBound`? we are not even doing a LIST call now?

Reply:
Renamed to CRC read bound